### DISCUSSION TITLE
MATEK H743 WING V1 : ACCELEROMETER ICM 20602 NOT OFFERED

### DISCUSSION THREAD

**Discussion by samsab1** [iNavFlight/inav-configurator#2219] [Q&A] [answered]
hello, i try to convert this FC to INAV 7.1.2 but it doesn't work: the list of accelerometer does'nt match what it is required: i mean ICM20602. when running betaflight this FC works, but uses ICM20602
![2024-10-26_07h42_08](https://github.com/user-attachments/assets/1c14439a-2807-4d27-a8f5-4c16fbddd2bf)

how can use this FC with INAV?
best regards from France

**Comment by sensei-hacker** [comment]
Matek switched the ICM20602 for the IMU_ICM42605. 

https://github.com/iNavFlight/inav/pull/7419/commits/8e0c57850dcc1a593a43c01a78ab8f8377fbb13b

Try choosing that. Or just use the MPU6500.

**Comment by samsab1** [comment]
hello sensei
![image](https://github.com/user-attachments/assets/a64b6db7-8f09-43f5-b137-a12e8c19ef24)
None of these choices allow you to calibrate the FC

**Comment by samsab1** [comment]
![icm](https://github.com/user-attachments/assets/255d1c68-26bc-4782-871c-85d95d782c96)

**Comment by mmosca** [comment, marked as answer]
ICM20602 uses the MPU6500 driver.

**Comment by samsab1** [comment]
but MPU6500 seems to not satisfy  INAV 7.1.2...

**Comment by samsab1** [comment]
https://github.com/user-attachments/assets/dd50155c-8be6-46b0-b0cc-37a07f33af17

**Comment by 0crap** [comment]
Works fine here.
Maybe your FC is damaged?

![MATEKH743](https://github.com/user-attachments/assets/16938819-6e9e-4467-9701-aae8149cea05)

**Reply by samsab1** [reply]
CAN I GET YOUR CLI?

**Comment by samsab1** [comment]
so strange; it works on betaflight (so i suppose fc is 100% ok) 
but not on inav

**Comment by samsab1** [comment]
is your fc a MATEK H743 wing V1?

**Comment by 0crap** [comment]
Ahhh, I'm running old stuff, but yours seems older. :-)
I have the purple one, which seems to be a V2.
Sorry.
![Screenshot 2024-10-26 162124](https://github.com/user-attachments/assets/d432adb3-8fcb-4748-a81d-3b2a1f1752d7)
We do seem to share the MPU6000 tho.

Try old INAV configurators, starting with INAV 3.0, to see if you can pinpoint where your problem started.
Might help the devs...

**Reply by samsab1** [reply]
yes, mine is blue, certainly older than yours

**Comment by samsab1** [comment]
i think i will stay with betaflight for this FC....too bad
